src/test/test_model_callbacks.py

```diff
@@ -0,0 +1,143 @@
+import pytest
+from unittest.mock import patch, MagicMock
+import json
+from dash.exceptions import PreventUpdate
+from src.callbacks.execute import (
+    store_dialog_models_in_redis_on_continue, 
+    store_sidebar_models_in_redis_on_update
+)
+
+class TestModelCallbacks:
+    
+    @pytest.fixture
+    def mock_redis(self):
+        """Mock the Redis client"""
+        with patch('src.callbacks.execute.redis_client') as mock_client:
+            yield mock_client
+    
+    @pytest.fixture
```

Comment:
I'm wondering if some of these fixtures, which are shared across multiple scripts, should be defined in a separate module outside this file.